view/frontend/templates/js/replacejs.phtml

```diff
@@ -416,15 +420,6 @@ if (!$block->isEnabled()) return;
                 if (callbacks.success_url) location.href = callbacks.success_url;
             },
 
-            onCheckoutStart: function() {
```

Comment:
why remove?

Reply:
This was for debugging. It shows when the pop-up window opens.